# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../iproute2/compile.patch.musl
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- iproute2-4.20.0/tc/f_flower.c.vanilla	2019-02-06 16:59:43.089134304 +0000
+++ iproute2-4.20.0/tc/f_flower.c	2019-02-06 16:59:51.637134207 +0000
@@ -19,6 +19,7 @@
 #include <linux/ip.h>
 #include <linux/tc_act/tc_vlan.h>
 #include <linux/mpls.h>
+#include <linux/limits.h>
 
 #include "utils.h"
 #include "tc_util.h"
